### PR TITLE
Add steps about how to Remove Filterable Attribute Values with 0 Products from Layered Navigation

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request (PR) adds steps about how to Remove Filterable Attribute Values with 0 Products from Layered Navigation

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- no

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- no

## Affected documentation pages

_List HTML links for affected pages on <https://docs.magento.com/user-guide/>._

- https://docs.magento.com/user-guide/catalog/navigation-layered-filterable-attributes.html

whatsnew
Updated the [Filterable Attributes](https://docs.magento.com/user-guide/catalog/navigation-layered-filterable-attributes.html) topic to include steps for removing filterable attribute values with 0 products from the layered navigation.

### CODE REVIEW

File: src/catalog/navigation-layered-filterable-attributes.md [hunk 1]
```diff
@@ -60,3 +60,27 @@ The [Position field]({% link stores/attributes-product.md %}) will be grayed out
 To test the setting, visit your store and navigate to the category from the main menu. The selection of filterable attributes appears in the layered navigation of the category page.
 
 Search, filter, and review the displayed products.
+
+## Remove Filterable Attribute Values with 0 Products from Layered Navigation
```
Comment (leslietilling) [inline src/catalog/navigation-layered-filterable-attributes.md:64]:
```suggestion
## Remove filterable attribute values from layered navigation
```

File: src/catalog/navigation-layered-filterable-attributes.md [hunk 2]
```diff
@@ -60,3 +60,27 @@ The [Position field]({% link stores/attributes-product.md %}) will be grayed out
...
+
+Layered navigation includes filters for all available attribute values and their product counts, including those with zero (0) product matches as shown in below image:
```
Comment (leslietilling) [inline src/catalog/navigation-layered-filterable-attributes.md:66]:
```suggestion
Layered navigation includes filters for all available attribute values and their product counts, including those with zero (0) product matches (as shown in the following image).
```

File: src/catalog/navigation-layered-filterable-attributes.md [hunk 3]
```diff
@@ -60,3 +60,27 @@ The [Position field]({% link stores/attributes-product.md %}) will be grayed out
...
+
+   ![]({% link images/images/filterable-attributes-on-plp.png %}){: .zoom}
+   _Zero filters displaying_
+
+This makes it difficult for customers to select a preferred product, and there is no need to display attribute values ​​with 0 products in the front-end.
+
+Therefore, follow the next steps to remove Filterable Attribute Values with 0 Products from Layered Navigation:
```
Comment (leslietilling) [inline src/catalog/navigation-layered-filterable-attributes.md:73]:
```suggestion
You can use the following steps to remove filterable attribute values with 0 Products from the layered navigation:
```

File: src/catalog/navigation-layered-filterable-attributes.md [hunk 4]
```diff
@@ -60,3 +60,27 @@ The [Position field]({% link stores/attributes-product.md %}) will be grayed out
...
+
+1. On the _Admin_ sidebar, go to **Stores** > **Attributes** > **Product**.
```
Comment (leslietilling) [inline src/catalog/navigation-layered-filterable-attributes.md:75]:
```suggestion
1. On the _Admin_ sidebar, go to **Stores** > _Attributes_ > **Product**.
```

File: src/catalog/navigation-layered-filterable-attributes.md [hunk 5]
```diff
@@ -60,3 +60,27 @@ The [Position field]({% link stores/attributes-product.md %}) will be grayed out
...
+
+1. Browse or use filtered search to find an attribute in the list and open it in edit mode.
+
+1. Click on **Storefront Properties** under **Attribute Information** section.
```
Comment (leslietilling) [inline src/catalog/navigation-layered-filterable-attributes.md:79]:
```suggestion
1. Under _Attribute Information_ , click **Storefront Properties**.
```

File: src/catalog/navigation-layered-filterable-attributes.md [hunk 6]
```diff
@@ -60,3 +60,27 @@ The [Position field]({% link stores/attributes-product.md %}) will be grayed out
...
+
+   ![]({% link images/images/storefront-properties-tab.png %}){: .zoom}
+   _Attribute Information section_
+
+1. Select **Filterable (with results)** from the Use in Layered Navigation dropdown.
```
Comment (leslietilling) [inline src/catalog/navigation-layered-filterable-attributes.md:84]:
```suggestion
1. For **Layered Navigation**, choose `Filterable (with results)`.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Edits for style and clarity

**Comment by EvheniiaKolkhiieva** [comment]
Hi @leslietilling , thank you for your review. I Have implemented your suggestions , please approve this PR ☀️
Thank you!

File: src/catalog/navigation-layered-filterable-attributes.md (hunk 6 above)
Comment (EvheniiaKolkhiieva) [reply src/catalog/navigation-layered-filterable-attributes.md]: Done ✅

File: src/catalog/navigation-layered-filterable-attributes.md (hunk 5 above)
Comment (EvheniiaKolkhiieva) [reply src/catalog/navigation-layered-filterable-attributes.md]: Done ✅

File: src/catalog/navigation-layered-filterable-attributes.md (hunk 4 above)
Comment (EvheniiaKolkhiieva) [reply src/catalog/navigation-layered-filterable-attributes.md]: Done ✅

File: src/catalog/navigation-layered-filterable-attributes.md (hunk 3 above)
Comment (EvheniiaKolkhiieva) [reply src/catalog/navigation-layered-filterable-attributes.md]: Done ✅

File: src/catalog/navigation-layered-filterable-attributes.md (hunk 2 above)
Comment (EvheniiaKolkhiieva) [reply src/catalog/navigation-layered-filterable-attributes.md]: Done ✅

File: src/catalog/navigation-layered-filterable-attributes.md (hunk 1 above)
Comment (EvheniiaKolkhiieva) [reply src/catalog/navigation-layered-filterable-attributes.md]: Done ✅

File: src/catalog/navigation-layered-filterable-attributes.md [hunk 7]
```diff
@@ -60,3 +60,27 @@ The [Position field]({% link stores/attributes-product.md %}) will be grayed out
...
+
+This makes it difficult for customers to select a preferred product, and there is no need to display attribute values ​​with 0 products in the front-end.
+
+You can use the following steps to remove filterable attribute values with 0 Products from the layered navigation:
+
+1. On the _Admin_ sidebar, go to **Stores** > _Attributes_ > **Product**.
+
+1. Browse or use filtered search to find an attribute in the list and open it in edit mode.
+
+ Under _Attribute Information_ , click **Storefront Properties**.
```
Comment (leslietilling) [inline src/catalog/navigation-layered-filterable-attributes.md:79]: one more fix, I'll commit
```suggestion
 1. Under _Attribute Information_ , click **Storefront Properties**.
```

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**